test/test_blocks.py

```diff
@@ -279,6 +279,25 @@ def test_blocks_does_not_replace_keyword_literal():
     assert output[0]["value"] == "NO_VALUE"
 
 
+def test_blocks_returns_correct_output_dict_single_key():
+
+    with gr.Blocks() as demo:
+        num = gr.Number()
+        num2 = gr.Number()
+        update = gr.Button(value="update")
+
+        def update_values():
+            return {output: output.update(value=42)}
```

Comment:
I think this should be as follows:
```suggestion
        def update_values(inp):
            return {num2: gr.update(value=42)}
```

Reply:
Yep you're right it's technically a class method.  Both work and I sometimes inadvertently use it as an instance method cause of the faster autocomplete. Will make the change!